content/api/vision/coordinates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Coordinates
+---
+
+[Polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system) are represented by a distance from a set point and an angle from a set direction. [Cartesian coordinates](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) are ones that are defined using a unique set of values (typically _x_, _y_ and _z_) to specify a point in space.
+
+{{% notice note %}}
+All distances are measured in metres.
+{{% /notice %}}
+
+## Polar Coordinates
+The polar coordinates are taken from the position of the webcam.
+
+ - `distance_metres` - the distance from the camera to the marker.
+ - `rot_x_rad` / `rot_x_deg` - the angle of rotation in radians/degrees about the x axis. The x axis runs horizontally in front of the camera. A positive angle would be one where the marker is in front of the camera.
+ - `rot_y_rad` / `rot_y_deg` - the angle of rotation in radians/degrees about the y axis. The y axis runs vertically in front of the camera. A positive angle would be one where the marker is rotated clockwise about the y axis looking from the top of the marker.
+ 
+{{% notice info %}}
+For example, if `distance_metres` was `1`, `rot_x_deg` was `0`, and `rot_y_deg` was `45`, the marker would be 45 degrees to the right of the robot, exactly 1 metre away.
+{{% /notice %}}
```

Comment:
:+1:
For even more clarity, could we get a diagram here? (Maybe out of scope for this PR though)

Reply:
This happens in #91.